### ISSUE TITLE
PGnet检测效果差

### ISSUE THREAD

**Issue by shuihsaiu** [PaddlePaddle/PaddleOCR#10321]
训练命令：python -m paddle.distributed.launch --gpus '0,1,2,3' tools/train.py -c configs/e2e/e2e_r50_vd_pg.yml -o Global.pretrained_model=./pretrain_models/train_step1/best_accuracy  Global.load_static_weights=True
配置文件：
Global:
  use_gpu: True
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/pgnet_r50_vd_totaltext/0707/
  save_epoch_step: 10
  # evaluation is run every 0 iterationss after the 1000th iteration
  eval_batch_step: [ 0, 1000 ]
  cal_metric_during_train: False  #设置为true时报错
  pretrained_model: 
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img:
  infer_visual_type: CN # two mode: EN is for english datasets, CN is for chinese datasets   设置数据集为英文还是中文
  valid_set: partvgg # two mode: totaltext valid curved words, partvgg valid non-curved words   设置文本框为弯曲或非弯曲
  save_res_path: ./output/pgnet_r50_vd_totaltext/0707/predicts_pgnet.txt
  character_dict_path: ppocr/utils/rebar_dict.txt    #设置字典路径
  character_type: CN
  max_text_length: 50 # the max length in seq
  max_text_nums: 30 # the max seq nums in a pic
  tcl_len: 64

Architecture:
  model_type: e2e
  algorithm: PGNet
  Transform:
  Backbone:
    name: ResNet
    layers: 50
  Neck:
    name: PGFPN
  Head:
    name: PGHead
    character_dict_path: ppocr/utils/rebar_dict.txt # the same as Global:character_dict_path

Loss:
  name: PGLoss
  tcl_bs: 64
  max_text_length: 50 # the same as Global: max_text_length
  max_text_nums: 30 # the same as Global：max_text_nums
  pad_num: 283 # 字典长度

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 50
  regularizer:
    name: 'L2'
    factor: 0.0001

PostProcess:
  name: PGPostProcess
  score_thresh: 0.5
  mode: fast   # fast or slow two ways
  point_gather_mode: align # same as PGProcessTrain: point_gather_mode

Metric:
  name: E2EMetric
  mode: A   # two ways for eval, A: label from txt,  B: label from gt_mat
  gt_mat_dir:  ./train_data/total_text/gt  # the dir of gt_mat
  character_dict_path: ppocr/utils/rebar_dict.txt
  main_indicator: f_score_e2e

Train:
  dataset:
    name: PGDataSet
    data_dir: ./train_data/
    label_file_list: [./train_data/train.txt]
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - E2ELabelEncodeTrain:
      - PGProcessTrain:
          batch_size: 4  # same as loader: batch_size_per_card
          use_resize: True
          use_random_crop: False
          min_crop_size: 24
          min_text_size: 4
          max_text_size: 512
          point_gather_mode: align # two mode: align and none, align mode is better than none mode
      - KeepKeys:
          keep_keys: [ 'images', 'tcl_maps', 'tcl_label_maps', 'border_maps','direction_maps', 'training_masks', 'label_list', 'pos_list', 'pos_mask' ] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: True
    batch_size_per_card: 18
    num_workers: 16

Eval:
  dataset:
    name: PGDataSet
    data_dir: ./train_data/
    label_file_list: [./train_data/test.txt]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - E2ELabelEncodeTest:
      - E2EResizeForTest:
          max_side_len: 768
      - NormalizeImage:
          scale: 1./255.
          mean: [ 0.485, 0.456, 0.406 ]
          std: [ 0.229, 0.224, 0.225 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'image', 'shape', 'polys', 'texts', 'ignore_tags', 'img_id']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2

不论是训练模型的检测效果还是inference模型的检测效果都不好，横排的数字效果还可以，竖排（90°、180°、270°）漏检很多。

**Comment by shuihsaiu** [comment]
我的数据集是中文数据集，里面是不同角度文本的集合，我之前用两阶段的det的检测效果挺不错的。

**Comment by shiyutang** [comment]
e2e模型对弯曲文本都可以检测，如果不同方向有差异效果，那么可以训练一个文字方向器进行预处理。

**Comment by shuihsaiu** [comment]
> e2e模型对弯曲文本都可以检测，如果不同方向有差异效果，那么可以训练一个文字方向器进行预处理。 分类器需要修改类别后对最后的分类层进行微调：https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.6/configs/cls/ch_PP-OCRv3/ch_PP-OCRv3_rotnet.yml

如果是训练过程中loss一直在60-80之间，这种是什么问题呢？

**Comment by Stonesusu** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/29848394/01bf61d2-08e8-41a4-818c-234d112771d0)
请问有出现eval时报错问题吗/

**Comment by UserWangZz** [comment]
该issue长时间未更新，暂将此issue关闭，如有需要可重新开启。
